frontend/src/pages/external/RedirectErrorState.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  PageSection,
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateBody,
+  EmptyStateFooter,
+  EmptyStateActions,
+  Button,
+} from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { EitherOrNone } from '~/typeHelpers';
+
+type RedirectErrorStateProps = {
+  title?: string;
+  errorMessage?: string;
+} & EitherOrNone<
+  {
+    fallbackUrl: string;
+    fallbackText: string;
+  },
+  {
+    actions: React.ReactNode | React.ReactNode[];
+  }
+>;
```

Comment:
Why the two states? It's usually odd to give such a large escape hatch 🤔 Not inherently problematic -- but what is the cost of making an Action?

Reply:
Ah -- I see the comment, didn't read everything at first haha -- seems it's easy enough to do it. Either take an array of data and handle the rendering or give it to the caller... seems like little gain to just handle the two strings.

In the end, I think you want to control the rendering as we _always_ want to redirect, no? Not like support a full modal or something

Reply:
sure. I can switch it to just being the actions and no fallback url/text